packages/core/src/jobs/constants.ts

```diff
@@ -24,30 +14,37 @@ export enum Jobs {
   createDocumentLogJob = 'createDocumentLogJob',
   createEventJob = 'createEventJob',
   publishEventJob = 'publishEventJob',
+  runBatchEvaluationJob = 'runBatchEvaluationJob',
+  runDocumentForEvaluationJob = 'runDocumentForEvaluationJob',
+  runDocumentInBatchJob = 'runDocumentInBatchJob',
+  runDocumentJob = 'runDocumentJob',
+  runEvaluationJob = 'runEvaluationJob',
+  publishToAnalyticsJob = 'publishToAnalyticsJob',
+  runLiveEvaluationJob = 'runLiveEvaluationJob',
 }
 
 export const QUEUES = {
   [Queues.defaultQueue]: {
     name: Queues.defaultQueue,
     jobs: [
-      createProviderLogJob,
-      runBatchEvaluationJob,
-      runDocumentForEvaluationJob,
-      runDocumentInBatchJob,
-      runDocumentJob,
-      runEvaluationJob,
+      'createProviderLogJob',
+      'runBatchEvaluationJob',
+      'runDocumentForEvaluationJob',
+      'runDocumentInBatchJob',
+      'runDocumentJob',
+      'runEvaluationJob',
     ],
   },
   [Queues.eventsQueue]: {
     name: Queues.eventsQueue,
-    jobs: [publishEventJob, createEventJob, publishToAnalyticsJob],
+    jobs: ['publishEventJob', 'createEventJob', 'publishToAnalyticsJob'],
   },
   [Queues.eventHandlersQueue]: {
     name: Queues.eventHandlersQueue,
     jobs: Object.values(EventHandlers).flat(),
   },
   [Queues.liveEvaluationsQueue]: {
     name: Queues.liveEvaluationsQueue,
-    jobs: [runLiveEvaluationJob],
+    jobs: ['runLiveEvaluationJob'],
   },
 } as const
```

Comment:
Why? Isn't this the same?

Reply:
no because now we don't import those jobs here anymore, which previously caused circular dependency shenanigans in nextjs build